ignite/src/main/java/site/ycsb/db/ignite/IgniteAbstractClient.java

```diff
@@ -159,9 +182,10 @@ private Ignite getIgniteClientNode() throws DBException, IgniteCheckedException
   }
 
   private Ignite getEmbeddedServerNode() throws IOException {
-    if (!"kv".equalsIgnoreCase(accessMethod) && !"sql".equalsIgnoreCase(accessMethod)) {
+    if (!"kv".equalsIgnoreCase(accessMethod) && !"sql".equalsIgnoreCase(accessMethod)
+        && !"jdbc".equalsIgnoreCase(accessMethod)) {
```

Comment:
As an alternative (not mandatory), we could use something like that:

```java
if (!Set.of("kv", "sql", "jdbc").contains(accessMethod.toLowerCase())) {
```

For several values, it's shorter and simpler to understand, comparing to `equalsIgnoreCase` calls. In addition, we could extract a given set into variable/constant and reuse to form an error message below.

Reply:
thanks, updated